cdap-ui/bower.json

```diff
@@ -44,7 +44,9 @@
     "d3-tip": "0.6.7",
     "esprima": "2.0.0",
     "ngInfiniteScroll": "1.2.1",
-    "d3-timeline": "0.0.5"
+    "d3-timeline": "0.0.5",
+    "inView": "1.0.0",
```

Comment:
does inView comes prepackaged in angular-inview? we are not including inView in gulp file, so it doesn't seem like this dependency is needed?